styles.js

```diff
@@ -1,14 +1,229 @@
+    border: 1px solid transparent; 
+    border-radius: var(--border-radius);          
+    padding: 10px 15px;           
+    cursor: pointer;             
+    transition: background-color 0.3s ease; 
+    font-family: "Junge", sans-serif;
+
+    &:hover {
+      background-color: var(--color-button-hover-background); 
+    }
+  }
+
+
+  article {
+    background-color: var(--color-background-cards); 
+    border-radius: var(--border-radius);
+    box-shadow: var(--box-shadow);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 15px;
+    position: relative;   
+  }
+
+  form {
+    background-color: var(--color-background-cards);
+    border-radius: var(--border-radius);
+    box-shadow: var(--box-shadow);
+    padding: 20px;
+    margin: 20px;
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+
+    @media (min-width: 720px) {
+      width: 50%; 
+      margin: 20px auto 0 auto;
+    }
+  }
+
+
+  img {
+    border-radius: 50%;
+  }
+
+  hr {
+  border: none;
+  height: 1px;
+  background-color: var(--color-button-border); 
+  width: 66.66%; 
+  margin: 15px auto;
+}
+
+  header {
+    font-size: var(--header-font-size);
+    font-weight: var(--header-font-weight);
+    color: var(--header-color);
+    margin: var(--header-margin);
+    text-align: var(--header-text-align);
+    font-weight: bold;
+    font-family: "Playfair Display", sans-serif;
+  }
+
+    h1{
+    font-size: 1.5rem;
+    display: flex;
+    justify-content: center;
+    color: var(--color-text-primary);
+  }
+
+  h2 {
+    font-size: 1.3rem;
+    color: var(--color-text-primary);
+    display: flex;
+    justify-content: center;
+  }
+
+  h3 {
+    font-size: 1.1rem;
+    display: flex;
+    justify-content: center;
+    color: var(--color-text-primary);
+  }
+
+  footer {
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    background-color: var(--color-footer-background);
+    padding: 10px;
+    display: flex;
+    justify-content: space-around;
+  }
+
+  ul {
+  list-style: none;
+
+  display: grid;
+  gap: 20px;
+  grid-template-columns: 1fr;
+
+    @media (min-width: 720px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+  }
+
+  label {
+    display: flex;
+    gap: 5px;
+    font-size: 0.95rem;
+    color: var(--color-text-primary);
+  }
+
+  input {
+    padding: 8px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    font-size: 1rem;
+    transition: border-color 0.3s ease;
+
+    &:focus {
+      border-color: #007bff;
+      outline: none;
+    }
+  }
+
+  textarea {
+    padding: 8px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    font-size: 1rem;
+    transition: border-color 0.3s ease;
+    resize: none;
+
+    &:focus {
+      border-color: #007bff;
+      outline: none;
+    }
+  }
+
+  fieldset {
+    border: 1px solid #ddd;
+    border-radius: 4px;
+    padding: 10px;
+  }
+
+  legend {
+    font-size: 1rem;
+    color: (--color-text-primary);
```

Comment:
I think there is a var missing?

```suggestion
    color: var(--color-text-primary);
```